dev-tools/mage/crossbuild.go

```diff
@@ -182,32 +182,6 @@ func CrossBuild(options ...CrossBuildOption) error {
 	// Each build runs in parallel.
 	Parallel(deps...)
 
-	// // It needs to run after all the builds, as it needs the darwin binaries.
-	// if err := assembleDarwinUniversal(params); err != nil {
-	// 	return err
-	// }
```

Comment:
[Blocker]
Ideally it wouldn't be removed. We'll gt back to working on producing a darwin/universal binary and it'd be just commenting in these lines.